dataflow_test.py

```diff
@@ -498,7 +498,20 @@ def test_slice_downstream(spec):
     assert result == the_source[specslice.start : specslice.stop : specslice.step]
 
 
-#TODO: Write test slice_close_all
+def test_slice_close_all():
```

Comment:
As these are user-facing documentation tests, I think that it's important that the difference between `close_all = True/False` should be made clear, without the reader having to hunt around.